hw2/04-background.js

```diff
@@ -1 +1,39 @@
-// Add your code here
+const toggleButton = document.getElementById("toggleButton");
+const input = document.getElementById("number");
+const container = document.getElementById("container");
+
+getRandomNumber = (max) => {
+	return Math.floor(Math.random() * max);
+};
+
+changeBackgroundColor = (maxRGB = 255) => {
+	const red = getRandomNumber(maxRGB);
+	const green = getRandomNumber(maxRGB);
+	const blue = getRandomNumber(maxRGB);
+	const alpha = 0.5; //to dim the color value
+	container.style.backgroundColor = `rgba(${red}, ${green}, ${blue}, ${alpha})`;
+};
+
+let intervalDuration = setInterval(() => {
+	changeBackgroundColor();
```

Comment:
You can just call the setInterval method with your function like: setInterval(changeBackgroundColor, 3000) instead of using =>

Reply:
Thanks for suggesting this. I wrote it as a ES6 function!